README.md

```diff
@@ -27,3 +27,11 @@ make lint
 ```
```

Comment:
Is there a startup command for the web socket server and the application?
If so can we have it added to the readme please.